docker-compose-services/redis/README.md

```diff
@@ -49,20 +49,35 @@ ddev redis-cli
 
 * Run `ddev describe` to check the status and ports used by the service.
 
+
+## Framework Quickstarts
+
+* Frameworks require configuration to use the redis service.
+* You need to replace `<DDEV_SITENAME>` values with your project's site name. Hint: `ddev describe`
+
+## Drupal
+
+* install the [`redis` module](https://www.drupal.org/project/redis)
+
 ```shell
-$ ddev describe
-...
-├──────────┬──────┬─────────────────────────────────────────┬───────────────────────┤
-│ SERVICE  │ STAT │ URL/PORT                                │ INFO                  │
-├──────────┼──────┼─────────────────────────────────────────┼───────────────────────┤
-│ redis    │ OK   │ https://example.ddev.site               │                       │
-│          │      │ InDocker: ddev-example-redis:6379       │                       │
-│          │      │ Host: localhost:54710                   │                       │
+ddev composer require drupal/redis
+```
+
+* add setting information to `web/sites/default/settings.local.php`; remember to update '<DDEV_SITENAME>'
+
+```php
+# DDEV SERVICE: Redis
+$settings['redis.connection']['host']      = 'ddev-${DDEV_SITENAME}-redis';  // Your Redis instance hostname.
```

Comment:
I wasn't able to resolve `${DDEV_SITENAME}` locally even if I loaded settings.local.php after settings.ddev.php in my settings.php, but `$_ENV['DDEV_SITENAME']` (or `getenv('DDEV_SITENAME')` did work. My ddev version is v1.18.2